lib/cext/ruby/ruby.h

```diff
@@ -545,10 +547,6 @@ volatile VALUE *rb_gc_guarded_ptr_val(volatile VALUE *ptr, VALUE val);
 #define RB_GC_GUARD_PTR(ptr) rb_gc_guarded_ptr(ptr)
 #endif
```

Comment:
This whole section should be exactly like https://github.com/ruby/ruby/blob/v2_3_5/include/ruby/ruby.h#L554-L570
The two `#endif` seem suspicious, and also the extra one at the end.

Reply:
Will push a fix.